tools/wurf_options.py

```diff
@@ -63,6 +66,20 @@ def resolve(ctx):
 
 class WurfOptions(Options.OptionsContext):
 
+    def execute(self):
+
+        # Execute the context, this will go out and invoke the option(...)
+        # function defined in the wscript. We have some extra tools that
+        # also define the option(...) function to avoid having to call load
+        # in all wscripts to initialize those, we can do it here.
+
+        self.load('wurf_common_tools')
```

Comment:
Roger that. I will experiment a bit with the implicit loading. Functionally these changes should not make any real difference - in the sense you can choose to remove those lines. However, you can also keep them, our existing code should still work fine with this version.